test/expr/unary/keypath/keypath.swift

```diff
@@ -741,3 +741,17 @@ func testSyntaxErrors() { // expected-note{{}}
   _ = \A.a?;
   _ = \A.a!;
 } // expected-error@+1{{}}
+
+class SR_10146 {
+  let abc: Int = 1
```

Comment:
Maybe it would make sense to add `@objc` here because that's what is going to trigger the diagnostic.

Reply:
Done!